### DISCUSSION TITLE
Add blacklist regex to exclude containers by name

### DISCUSSION THREAD

**Discussion by elyulka** [neuroforgede/docker-engine-events-exporter#2] [Ideas]
It would be great to exclude noisy processes like swarm-cronjob spawning every minute laravel cron from taking prometheus resources.

**Comment by s4ke** [comment]
This sounds like a good idea. Combining this with garbage collection (see https://github.com/neuroforgede/docker-engine-events-exporter/issues/1) should yield a better situation.

We are open to contributions. Want to take this up?
